common/beerocks/bwl/include/bwl/base_wlan_hal_types.h

```diff
@@ -215,6 +215,11 @@ typedef struct {
     sClientAssociationParams params;
 } sACTION_APMANAGER_CLIENT_ASSOCIATED_NOTIFICATION;
 
+typedef struct {
```

Comment:
Please include a Doxygen comment for each new class/struct.

Reply:
We really should ban the use of "typdef". On cpp it is not needed.

Reply:
Both comments are correct; however:
- the whole file currently has no doxygen comments;
- it's pretty obvious what it does/means, so a doxygen comment doesn't help much;
- the whole file currently has typedefs;
- this will most likely be refactored for M3.

Therefore, we're not going to bother with fixing this.